### ISSUE TITLE
tags, to v or not to v?

### ISSUE THREAD

**Issue by geemus** [superfly/flyctl-actions#59]
Hey, just wanted to call this out since it has caught me out a couple of times now.

Most of the `vX.Y` releases actually have tags that just look like `X.Y` (ie they don't have the leading v like their matching release does). There are a few exceptions (v1 and v1.4 for instance), but that led to me just bumping from `v1.4` (which does exist) to `v1.5` (which doesn't exist, although `1.5` does).

It would be helpful if this were more consistent. I think superfly might be the only action that I interact with regularly that doesn't use v prefixes consistently, which I suspect also makes it all the more likely that I would make this kind of mistake.

Any way that this could be resolved to be more consistent and/or that future releases have both kinds of tags if you want to go that route?

Thanks.

**Comment by thewilkybarkid** [comment]
This caught me out too a while back: https://github.com/superfly/flyctl-actions/issues/24#issuecomment-1543636695.

**Comment by jaydenseric** [comment]
There is `v1.4`, but for v1.5 you have to do just `1.5`. The tags are a sick joke:

https://github.com/superfly/flyctl-actions/tags

Look at all the variations for v1:

- https://github.com/superfly/flyctl-actions/releases/tag/v1
- https://github.com/superfly/flyctl-actions/releases/tag/1
- https://github.com/superfly/flyctl-actions/releases/tag/1.0

All 3 point to different commits, and have different titles and descriptions! What is going on???

**Comment by Jokero** [comment]
This repo seems to be abandoned

**Comment by lucianthorr** [comment]
Yeah, this is both crazy confusing and probably abandoned.  I'm new to Fly.io so this makes for a weird first impression.
